doc/response-tagging.rst

```diff
@@ -20,29 +20,28 @@ Setup
 
     Make sure to :doc:`configure your proxy <proxy-configuration>` for tagging first.
 
-The tag handler is a decorator around the ``CacheInvalidator``. After
-:doc:`creating the invalidator <cache-invalidator>` with a proxy client
-that implements the ``BanInterface``, instantiate the ``TagHandler``::
+The response tagger is a decorator around a proxy client that implements
+the ``TagsInterface``, handling tags for the current response::
 
-    use FOS\HttpCache\Handler\TagHandler;
+    use FOS\HttpCache\ResponseTagger;
 
-    // $cacheInvalidator already created as instance of FOS\HttpCache\CacheInvalidator
-    $tagHandler = new TagHandler($cacheInvalidator);
+    // $proxyClient already created, implementing FOS\HttpCache\ProxyClient\Invalidation\TagsInterface
+    $responseTagger = new ResponseTagger($proxyClient);
 
 Usage
 ~~~~~
 
 With tags you can group related representations so it becomes easier to
 invalidate them. You will have to make sure your web application adds the
-correct tags on all responses. You can add tags to the handler using::
+correct tags on all responses. You can add tags to the response using::
```

Comment:
ah, perfect. but lets talk about the ResponseTagger here, not about adding to the response, because that could lead to assume that the tags would magically happen to be in the resposne without doing anything :-)